Java/BST.java

```diff
@@ -0,0 +1,349 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/** A class to denote a node of the BST.
+ */
+class Node {
+    /** AVL Node hold's this value.
+     */
+    public int value;
+
+    /** Pointer to left child of the BST Node.
```

Comment:
Use // instead of /**/ for single line comments.
Note: the comment is redundant and can be safely removed.